### PR TITLE
Feature/fix ssl library

### PR DESCRIPTION
**Pull request by olivierbrand**

### CODE REVIEW

**Comment by olivierbrand** [comment]
new ssl library

File: PurchaseController/Classes/ReceiptValidation/LocalReceiptValidatorImplementation.swift [hunk 1]
```diff
@@ -436,11 +436,17 @@ fileprivate struct ReceiptParser {
             return nil
         }
         
-        let integer = c2i_ASN1_INTEGER(nil, &intPointer, intLength)
-        let result = ASN1_INTEGER_get(integer)
-        ASN1_INTEGER_free(integer)
+//         let integer = c2i_ASN1_INTEGER(nil, &intPointer, intLength)
```
Comment (denystrush) [inline PurchaseController/Classes/ReceiptValidation/LocalReceiptValidatorImplementation.swift:439]: Please remove unused commented code

**Review by denystrush — APPROVED** [review]

**Comment by denystrush** [comment]
minor comment

File: PurchaseController/Classes/ReceiptValidation/LocalReceiptValidatorImplementation.swift (hunk 1 above)
Comment (olivierbrand) [reply PurchaseController/Classes/ReceiptValidation/LocalReceiptValidatorImplementation.swift:439]: Please check new updated PR, I removed the commented code

**Pull request merged**